tests/neg/singletonOrs.scala

```diff
@@ -1,6 +1,6 @@
 object Test {
    def foo: 1 | 2 = 1 // error // error
    def bar: 3 | 4 = foo // error // error
-   def foo: 1 | 2 = 1 // error // error
-   def bar: 1 = foo
+   def foo: 1 | 2 = 1 // error // error // error
+   def bar: 1 = foo // error
```

Comment:
Actually I think a better solution is to rename the last two methods. This test is not meant to test duplicate definition